homeassistant/components/tesla_fleet/strings.json

```diff
@@ -412,6 +412,18 @@
       "vehicle_state_odometer": {
         "name": "Odometer"
       },
+      "island_status": {
+        "name": "Grid Status",
+        "state": {
+          "island_status_unknown": "Unknown",
```

Comment:
If the value is unknown, the entity should return `None`, otherwise the entity knows 2 "Unknown" states, and that is bad

Reply:
So when the sensor returns "island_status_unknown" I should instead return `None`? I'll add a lambda.

Reply:
Yep